### ISSUE TITLE
Reading file from google storage using async call gives error

### ISSUE THREAD

**Issue by mishaetaya** [GoogleCloudPlatform/nodejs-docs-samples#364]
Hello,

I've been trying to make the asynchronous call `startRecognition(gcsUri, config)` but getting the following error: `Error: Requested entity was not found.`

my gcsUri is `gs://bucket/file.wav`

```js
config = {encoding: 'LINEAR16',
	  sampleRateHertz: 8000,
	  languageCode: 'en-US'}
```

Any guidance would be appreciated.

Thank you

best

**Comment by jmdobry** [comment]
Is your gcsUri really `gs://bucket/file.wav`, or did you change out the real names for placeholder names?

**Comment by mishaetaya** [comment]
apologies, yes those are placeholders indeed (not real names).  It started to work actually, but all I did differently was to comment out the projectId in 

var gcs = require('@google-cloud/storage')({
   // projectId: process.env.GCLOUD_PROJECT,
  credentials: require('key.json')
});

... and I originally didn't have 'transcription' defined as a const because it was defined as a 'var' further up in the file. I changed it to be defined in the scope of the startRecognition function.  I'm glad that it is working, but I don't really understand why these two changes resolved the error, or what that error was.
